### PR TITLE
Updates polyline position/scale when GPS location is updated

### PR DESCRIPTION
**Pull request by Pilot-Marc**
### When GPS location (coordinates or altitude) changes, apply the same scaling logic to Polylines as we already do to Annotation nodes

This allows polylines to scale on par with annotation nodes when the location or altitude changes rapidly, such as in an automobile or elevator.

### Breaking Changes
Polyline is now a subclass of LocationNode (as LocationAnnotationNode node already is).  This should probably not break anything, but we should be on the lookout.  

### Meta
- Tied to Version Release(s):

### Checklist

- [x] Appropriate label has been added to this PR (i.e., Bug, Enhancement, etc.).
- [ ] Documentation has been added to all `open`, and `public` scoped methods and properties.
- [x] Changelog has been updated
- [ ] Tests have have been added to all new features. (not a requirement, but helpful)
- [ ] Image/GIFs have been added for all UI related changed.

### Screenshots

### CODE REVIEW

File: Sources/ARKit-CoreLocation/Nodes/PolylineNode.swift [hunk 1]
```diff
@@ -34,9 +34,15 @@ public class PolylineNode {
         self.altitude = altitude
         self.boxBuilder = boxBuilder ?? Constants.defaultBuilder
 
+		super.init(location: nil)
```
Comment (intere) [inline Sources/ARKit-CoreLocation/Nodes/PolylineNode.swift]: Can you remove the extra whitespace here?

**Comment by intere** [comment]
Also - there are merge conflicts that need to be resolved.  Let me know if you need help with it.

File: Sources/ARKit-CoreLocation/Nodes/PolylineNode.swift (hunk 1 above)
Comment (Pilot-Marc) [reply Sources/ARKit-CoreLocation/Nodes/PolylineNode.swift]: I see you beat me to it.  I'll resolve conflicts, thanks!

**Comment by intere** [comment]
Don't worry about the broken CI - I'll be fixing that.

**Pull request merged**